docs/modules/ROOT/pages/components.adoc

```diff
@@ -283,6 +283,115 @@ mod MyContract {
 }
 ----
 
+=== Immutable Config
```

Comment:
Should we name this "Immutable configuration" instead or do you think it's worth keeping the capitalization and abbreviating "configuration" the same as it's defined in the shamans proposal?

Reply:
I'm going to create a SNIP proposal with the name Immutable Component Config. Do you think is better to use Configuration instead of Config (as a native English speaker I'm leaning toward your preferred approach).

Reply:
Either way works. Personally, I'd write out the full name when introducing it and then use the abbreviated term. Though, if that's the SNIP name, then it's probably better to keep it consistent with the SNIP